css/style.css

```diff
@@ -245,3 +258,119 @@ main {
   font-size: 40px;
   line-height: 52px;
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  border-radius: 100px 0 0 0;
+  background-image: url(../img/contact-shape.svg);
+  background-position: top 20px right;
+  width: 375px;
+  height: 779px;
+  margin-top: 103px;
+  padding-bottom: 20px;
+}
+
+.contact-text {
+  padding-top: 103px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 24px;
+}
+
+.contact-text h2 {
+  text-align: center;
+  color: #fff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.contact-text p {
+  margin-top: 14px;
+  text-align: center;
+  color: #ebebff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+input[type="text"] {
+  width: 327px;
+  height: 48px;
+  background-color: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+input[type="email"] {
+  margin-top: 20px;
+  width: 327px;
+  height: 48px;
+  background-color: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+textarea {
+  margin-top: 18px;
+  width: 327px;
+  height: 170px;
+  background-color: #fff;
+  padding: 15px 38px 15px 16px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 20px;
+  box-sizing: border-box;
+}
+
+.button-submit {
+  margin-top: 22px;
+  width: 170px;
+  height: 48px;
+  background-color: #fff;
+  border-radius: 8px;
+  border: none;
+  padding: 12px 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  cursor: pointer;
+}
+
+.button-submit:hover,
+.button-submit:active {
+  background-color: #e5e5e5;
+  color: #000;
+}
```

Comment:
- [ ] The hover and active colors are different in your code from design, Please apply the hover and active colors from the figma template and make sure it matches the design.

Check this image for reference

![Screenshot (105)](https://user-images.githubusercontent.com/83803414/154108297-23bc71dc-9641-422a-bcb7-f69dba818664.png)